package.json

```
@@ -4,7 +4,7 @@
   "description": "Run Next.js seamlessly on Netlify",
   "scripts": {
     "build:demo": "cd demos/default && npm run build",
-    "cy:open": "cypress open --config-file cypress/config/all.json",
+    "cy:open": "cypress open --config-file=cypress/config/all.config.ts",
```

Since Cypress now also has component testing, `cypress open` will put one into a screen where you have to choose e2e and your browser first.

You can short-circuit it like this:

```suggestion
    "cy:open": "cypress open --config-file=cypress/config/all.config.ts --browser chrome --e2e",
```